Helper/Order.php

```diff
@@ -981,10 +981,13 @@ public function deleteOrderByIncrementId($displayId)
      * @param $orderIncrementId
      * @return OrderModel|false
      */
-    protected function getExistingOrder($orderIncrementId)
+    public function getExistingOrder($orderIncrementId)
     {
         /** @var OrderModel $order */
-        return $this->cartHelper->getOrderByIncrementId($orderIncrementId, true);
+        return $this->cartHelper->getOrderByIncrementId($orderIncrementId, true) ?:
+            // bypass missing increment id in PPC transaction data - themporary fix.
+            // TODO: remove
```

Comment:
How temporary is this? When will this TODO be actioned? Can we create an Asana task to remove this so it doesn't become forgotten?

Reply:
@vitaliyreznikov created a task, https://app.asana.com/0/941920570700290/1159061667010227/f. Anyway, this is not urgent, can stay for a while since it does not affect the regular functionality.